File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GoogleMap/Features/MapAddressPropertyPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.GoogleMap.Features
{
    public partial class MapAddressPropertyPage : UserControl
    {
        public MapAddressPropertyPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Click for Button control
        /// </summary>
        private void mobjSetAddressButton_Click(object sender, EventArgs e)
        {
            // Set map address for GoogleMap control
            this.mobjGoogleMap.MapAddress = mobjAddressTextBox.Text;
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GoogleMap/Features/MapAddressPropertyPage.vb
Namespace CompanionKit.Controls.GoogleMap.Features

	Public Class MapAddressPropertyPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles Click for Button control
        ''' </summary>
        Private Sub mobjSetAddressButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjSetAddressButton.Click
            ' Set map address for GoogleMap control
            Me.mobjGoogleMap.MapAddress = mobjAddressTextBox.Text
        End Sub
    End Class

End Namespace